### PR TITLE
Run kumade:post_deploy if deployment succeeds.

### PR DESCRIPTION
**Pull request by hoverlover**

### CODE REVIEW

File: lib/kumade/deployer.rb [hunk 1]
```diff
@@ -66,5 +71,9 @@ def ensure_heroku_remote_exists
     def run_predeploy_task
       RakeTaskRunner.new("kumade:pre_deploy").invoke
     end
+
+    def run_postdeploy_task
```
Comment (gabebw) [inline lib/kumade/deployer.rb]: Can you rename this to `run_post_deploy_task`? It looks odd having "post_deploy" with the underscore used everywhere but in this method.

**Comment by gabebw** [comment]
This looks like a great addition! I have just one comment, then it'll be good to merge.

File: lib/kumade/deployer.rb (hunk 1 above)
Comment (hoverlover) [inline lib/kumade/deployer.rb]: Yeah, I can do that.  I don't like it either, but I was just following convention with `run_predeploy_task`.  You still want me to change it?

Comment (gabebw) [inline lib/kumade/deployer.rb]: Thanks for following convention. I would still like it changed, yeah. `run_predeploy_task` should probably be changed too, but that's outside the scope of this pull.

Comment (hoverlover) [inline lib/kumade/deployer.rb]: OK, I'll get it changed.

**Pull request merged**